src/core/public/overlays/flyout/__snapshots__/flyout_service.test.tsx.snap

```diff
@@ -53,6 +53,7 @@ Array [
               Array [
                 Object {
                   "children": <CoreThemeProvider
+                    globalStyles={false}
```

Comment:
Q: Is this correct to be `false` if the context provider defaults to `true`? Is it because the tests don't use the `CoreContextProvider` set by the rendering service?

Should we add tests to make sure that the components rendered by the rendering service have this parameter set to `true`?

Reply:
> Is this correct to be false if the context provider defaults to true

The `CoreContextProvider` is used in 4 mount-point services, but we only want the `rendering_service` service to inject global styles. The underlying EUI component defaults to injecting global styles because 99% of consuming applications will only ever have one instance of said component. So for Kibana, I've inverted the logic to _not_ default to injecting global styles and instead configure only the rendering service to inject global styles.

> Is it because the tests don't use the CoreContextProvider set by the rendering service?

Yes. This test only mounts the provider in the `flyout_service`.

> Should we add tests to make sure that the components rendered by the rendering service have this parameter set to true?

I can add this 👍

Reply:
Added a test: 5fc72a3
Updated the comment for clarity: 603efca